app/models/concerns/twitter_user/api.rb

```diff
@@ -0,0 +1,279 @@
+      ]
+      # drilldown_series = [
+      #   {name: 'good', id: 'good', data: items.slice(0, good - 1).map { |i| [i.screen_name, i.score] }},
+      #   {name: 'not_so_bad', id: 'not_so_bad', data: items.slice(good, not_so_bad - 1).map { |i| [i.screen_name, i.score] }},
+      #   {name: 'so_so', id: 'so_so', data: items.slice(not_so_bad, so_so - 1).map { |i| [i.screen_name, i.score] }},
+      # ]
+    end
+
+    def one_sided_following_graph
+      friends_size = friends.size
+      one_sided_size = one_sided_following.size
+      [
+        {name: I18n.t('legend.one_sided_following'), y: (one_sided_size.to_f / friends_size * 100)},
+        {name: I18n.t('legend.others'), y: ((friends_size - one_sided_size).to_f / friends_size * 100)}
+      ]
+    end
+
+    def one_sided_followers_graph
+      followers_size = followers.size
+      one_sided_size = one_sided_followers.size
+      [
+        {name: I18n.t('legend.one_sided_followers'), y: (one_sided_size.to_f / followers_size * 100)},
+        {name: I18n.t('legend.others'), y: ((followers_size - one_sided_size).to_f / followers_size * 100)}
+      ]
+    end
+
+    def mutual_friends_rate
+      friendship_size = friends_count + followers_count
+      return [0.0, 0.0, 0.0] if friendship_size == 0
+      [
+        mutual_friends.size.to_f / friendship_size * 100,
+        one_sided_following.size.to_f / friendship_size * 100,
+        one_sided_followers.size.to_f / friendship_size * 100
+      ]
+    end
+
+    def mutual_friends_graph
+      rates = mutual_friends_rate
+      sliced = rates[0] < 25
+      [
+        {name: I18n.t('legend.mutual_friends'), y: rates[0], sliced: sliced, selected: sliced},
+        {name: I18n.t('legend.one_sided_following'), y: rates[1]},
+        {name: I18n.t('legend.one_sided_followers'), y: rates[2]}
+      ]
+    end
+
+    def common_friends_graph(other)
+      friends_size = friends.size
+      common_friends_size = common_friends(other).size
+      [
+        {name: I18n.t('legend.common_friends'), y: (common_friends_size.to_f / friends_size * 100)},
+        {name: I18n.t('legend.others'), y: ((friends_size - common_friends_size).to_f / friends_size * 100)}
+      ]
+    end
+
+    def common_followers_graph(other)
+      followers_size = followers.size
+      common_followers_size = common_followers(other).size
+      [
+        {name: I18n.t('legend.common_followers'), y: (common_followers_size.to_f / followers_size * 100)},
+        {name: I18n.t('legend.others'), y: ((followers_size - common_followers_size).to_f / followers_size * 100)}
+      ]
+    end
+
+    def usage_stats_graph
+      client.usage_stats(__uid_i, tweets: statuses)
+    end
+
+    def frequency_distribution(words)
+      words.map { |word, count| {name: word, y: count} }
+    end
+
+    def clusters_belong_to_cloud
+      clusters_belong_to.map.with_index { |(h, c), i| {text: h, size: c, group: i % 20} }
```

Comment:
__[rubocop]__
* Space inside { missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)
* Line is too long. [89/80] [:ref](https://github.com/bbatsov/ruby-style-guide#80-character-limits)
* Space inside } missing. [:ref](https://github.com/bbatsov/ruby-style-guide#spaces-operators)